site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -121,9 +121,17 @@ relinquish control to an external service which likely doesn't have as much info
 {{< /alert >}}
 
 #### Shutdown()
-This tells Agones to shut down the currently running game server.
-The GameServer state will be set `Shutdown` and the 
-backing Pod will be deleted, if they have not shut themselves down already. 
+This tells Agones to shut down the currently running game server. The GameServer state will be set `Shutdown` and the 
+backing Pod will be Terminated.
+
+It's worth reading the [Termination of Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
+Kubernetes documentation, to understand the termination process, and the related configuration options.
+
+As a rule of thumb, implement a graceful shutdown in you game sever process when it receives the TERM signal
```

Comment:
Minor typo: `shutdown in you game sever`; should be "your"

Reply:
:facepalm: Thanks!